esbuild.mjs

```
@@ -25,24 +32,48 @@ const webviewBuildOptions = {
+  const [, webviewResult] = await Promise.all([
+    build(extensionBuildOptions),
+    build(webviewBuildOptions),
+  ]);
+  const budget = assertInitialWebviewBudget(webviewResult.metafile);
+  writeBundleMetadata({
+    metafile: webviewResult.metafile,
+    budget,
+    outputPath: bundleMetadataPath,
+    production: isProduction,
+  });
+  console.log(
```

ESLint

[error] Unexpected console statement.

(no-console)